examples/mongodbatlas_advanced_cluster/global-cluster/main.tf

```
@@ -44,8 +43,58 @@ resource "mongodbatlas_advanced_cluster" "cluster" {
       priority      = 6
       region_name   = "US_EAST_2"
     }
+  }
+
+  replication_specs { # shard 2 - zone n1
+    zone_name = "zone n1"
+
+    region_configs {
+      electable_specs {
+        instance_size = "M10"
+        node_count    = 3
+      }
+      analytics_specs {
+        instance_size = "M10"
+        node_count    = 1
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "US_EAST_1"
+    }
+
+    region_configs {
+      electable_specs {
+        instance_size = "M10"
+        node_count    = 2
+      }
+      analytics_specs {
+        instance_size = "M10"
+        node_count    = 1
+      }
+      provider_name = "AZURE"
+      priority      = 6
+      region_name   = "US_EAST_2"
+    }
+  }
 
-    region_configs { # shard n3
+  replication_specs { # shard 1 - zone n2
```

```suggestion
  replication_specs { # shard 3 - zone n2
```
Same for `# shard 2 - zone n2` -> `# shard 4 - zone n2`
Do you think this would make it more clear? At the end the configuration is defining 4 shards, 2 in each zone.

---

I think since the zone is mentioned alongside, this should not be updated to shard 3 & 4. In the current way it seems clear that it's 2 shards in each zone